### ISSUE TITLE
"Go to album" and "More of artist" not showing while tapping player menu button

### ISSUE THREAD

**Issue by AnyoneMouse** [fast4x/RiMusic#809]
### Steps to reproduce the bug

I was thinking about this bug and decided to check how do I reproduce it.
So I tried playing songs from different places.

At first while playing from playlists those mentioned **options** (go to album, more of artist) didn't appear.
If I'm not clear enough about the **options** I mean these:
![1708602449040](https://github.com/fast4x/RiMusic/assets/85461823/d6083275-9bed-48b5-af05-bab975d2d9c7)

Then I played by shuffling artist. I mean this: 
![1708602511208](https://github.com/fast4x/RiMusic/assets/85461823/558a2fbf-b055-4739-af2c-4b48f18e7bad)

and those **options** then appeared. 
Then again I tried playing from "singles" and those **options** didn't appear. 

Again I tried playing a song from "albums" and played same song while shuffling artist. In the case of albums those **options** didn't appear but those appeared in artist shuffling.

Just now I tried playing a song through "online search" feature and this time those **options** appeared.

In short, except "artist shuffling" and "online search" feature those **options** aren't appearing at all in any other case.
Though I it is possible I missed anything but that's all I've come up with.

### Expected behavior

Appearing in all cases.

### Actual behavior

Didn't appear in most cases.

### RiMusic version

0.6.22

### Android version

Android 11

### Additional information

Please fix this bug those options were really helpful. At least the "More of artist" one.

Or wait am I actually missing something? Were those filtered out intentionally?

**Comment by fast4x** [comment]
I'll check It all, thanks for details

**Comment by siggi1984** [comment]
> ### Steps to reproduce the bug
> 
> I was thinking about this bug and decided to check how do I reproduce it. So I tried playing songs from different places.
> 
> At first while playing from playlists those mentioned **options** (go to album, more of artist) didn't appear. If I'm not clear enough about the **options** I mean these: ![1708602449040](https://private-user-images.githubusercontent.com/85461823/306983189-d6083275-9bed-48b5-af05-bab975d2d9c7.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg2MTU0OTEsIm5iZiI6MTcwODYxNTE5MSwicGF0aCI6Ii84NTQ2MTgyMy8zMDY5ODMxODktZDYwODMyNzUtOWJlZC00OGI1LWFmMDUtYmFiOTc1ZDJkOWM3LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIyVDE1MTk1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdjNzRhZDUxOThjMTA2ZWY1ZDQ0ODM2MzkyNjJjYzI0MWVjNTkwOGM4ZDAwZDZjZmY1MDEzNDAyOTU1Y2UyMDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1Fc4dxJ_N1M461j321vOfOpkLuByepM0NtPQmK1cQsQ)
> 
> Then I played by shuffling artist. I mean this: ![1708602511208](https://private-user-images.githubusercontent.com/85461823/306983444-558a2fbf-b055-4739-af2c-4b48f18e7bad.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg2MTU0OTEsIm5iZiI6MTcwODYxNTE5MSwicGF0aCI6Ii84NTQ2MTgyMy8zMDY5ODM0NDQtNTU4YTJmYmYtYjA1NS00NzM5LWFmMmMtNGI0OGYxOGU3YmFkLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIyVDE1MTk1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRhMGUxMDM2ZjA3NjQ2MWUxNjFiNDQ3NWQ2ZTg1NDAzYTBhMGYyOTU3N2UzYWRmODk2MDQwNDdiMjI5Y2E2NGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ZMaR5XrVScKfu1-FAreijKGQ4_MXgppka_Jy8-wuOUs)
> 
> and those **options** then appeared. Then again I tried playing from "singles" and those **options** didn't appear.
> 
> Again I tried playing a song from "albums" and played same song while shuffling artist. In the case of albums those **options** didn't appear but those appeared in artist shuffling.
> 
> Just now I tried playing a song through "online search" feature and this time those **options** appeared.
> 
> In short, except "artist shuffling" and "online search" feature those **options** aren't appearing at all in any other case. Though I it is possible I missed anything but that's all I've come up with.
> ### Expected behavior
> 
> Appearing in all cases.
> ### Actual behavior
> 
> Didn't appear in most cases.
> ### Screenshots/Screen recordings
> 
> _No response_
> ### Logs
> 
> _No response_
> ### RiMusic version
> 
> 0.6.22
> ### Android version
> 
> Android 11
> ### Additional information
> 
> Please fix this bug those options were really helpful. At least the "More of artist" one.
> 
> Or wait am I actually missing something? Were those filtered out intentionally?

The same for me

**Comment by fast4x** [comment]
Fixed, available from version 0.6.23

**Comment by faustaahmad** [comment]
More of artist doesnt appear in Songs tab too, like op said that it doesnt appear on most of the time

**Comment by fast4x** [comment]
Just fixed in 0.6.23
